src/state/room/roomReducer.ts

```diff
@@ -256,6 +256,11 @@ const roomReducer: Reducer<RoomState> = (state: RoomState = initialState, anyAct
 					areVisible: false,
 					areSimple: false
 				},
+				chat: {
```

Comment:
GameStateCleared is also sent when round is changed. That will cause chat to disappear on new round. That is unexpected behavior

Reply:
Ok. Will move this in separate action.